discovery-provider/src/tasks/index_aggregate_track.py

```diff
@@ -0,0 +1,167 @@
+import logging
+
+from src.tasks.celery_app import aggregate_worker, celery, celery_worker
+from src.tasks.index_aggregate_user import get_latest_blocknumber
+
+logger = logging.getLogger(__name__)
+
+AGGREGATE_TRACK = "aggregate_track"
+
+# UPDATE_AGGREGATE_TRACK_QUERY
+# Get a higher bound blocknumber to check for new save/repost activity
+# Find a subset of tracks that have changed after that blocknumber
```

Comment:
In this case, only the lower bound matters. Similar to aggregate_user, it's ok to recount some newer items but we don't want to recount the same old items every time. This is what we want: "find a subset of tracks that have changed after that blocknumber."